### DISCUSSION TITLE
Using qodana docker images in a corporate environment with its own root CA certificate

### DISCUSSION THREAD

**Discussion by guymahieu** [JetBrains/Qodana#171] [Q&A]
I'm trying to get the qodana-jvm-community docker image to work for our environment where a corporate root certificate is used for the nexus repository https connection.

I can create my own docker image FROM jetbrains/qodana-jvm-community and add our certificate to the cacerts of the jre used (under /opt/idea/jbr in the image I guess).

Would that be the best way to go about it?

**Comment by brichbash** [comment]
Hello,

It does look like a good option. But please remember that Maven/Gradle may use a JDK different from what IntelliJ IDEA in Qodana runs on. So you might have to pass the certificate to both JREs. You can explicitly tell Qodana to use `jbr-17` by adding `projectJDK: jbr-17` to `qodana.yaml` if `jbr-17` suits your project. In that case, adding your certificate to `/opt/idea/jbr`
